apiserver/clusterservice/scaleimpl.go

```diff
@@ -292,49 +291,48 @@ func ScaleDown(deleteData bool, clusterName, replicaName, ns string) msgs.ScaleD
 		return response
 	}
 
-	//if this was the last replica then remove the replica service
-	var replicaList *v1.DeploymentList
-	selector := config.LABEL_PG_CLUSTER + "=" + clusterName + "," + config.LABEL_SERVICE_NAME + "=" + clusterName + "-replica"
-	replicaList, err = kubeapi.GetDeployments(apiserver.Clientset, selector, ns)
+	// selector in the format "pg-cluster=<cluster-name>,role=replica"
+	// which will grab any/all replicas
+	selector := fmt.Sprintf("%s=%s,%s=%s", config.LABEL_PG_CLUSTER, clusterName,
+		config.LABEL_PGHA_ROLE, "replica")
```

Comment:
We probably should make `"replica"` a constant in one of the global config files, perhaps the catch-all `config/labels.go`

Reply:
Since this was a back-patch, I just grabbed this change as it exists on the Master branch.  Agree a constant would be better though, so I can add that.  We'll prob just want to circle back and do the same on Master as some point.

Reply:
Yup that makes sense.